### PR TITLE
[SEASK-27] Quick Start -> Introductions to APIs (via Postman)

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: sample-api-calls/SEA Modernization Starter Kit Environment (Local).postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+	"id": "e50e829a-4d3d-4a97-80c5-f7ecb1825fbf",
+	"name": "SEA Modernization Starter Kit",
+	"values": [
+		{
+			"key": "ApiBaseUrl",
+			"value": "https://api-stage.ed-fi.org/v5.2/api",
```
Comment (vimayya) [inline sample-api-calls/SEA Modernization Starter Kit Environment (Local).postman_environment.json]: @semalaiappan could you change this to https://localhost/WebApi

File: sample-api-calls/SEA Modernization Starter Kit.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,2486 @@
...
+		{
+			"listen": "prerequest",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					"CreateAccessToken(\"TokenExpiry\",\"AccessToken\",\"ApiKey\",\"ApiSecret\")",
+					"",
+					"// Adapted from: https://marcin-chwedczuk.github.io/automatically-generate-new-oauth2-tokens-when-using-postman",
+					"// Assumes Environment with Environment Variables: ApiBaseUrl, ApiKey, ApiSecret",
+					"// See https://gist.github.com/blmeyers/21138bbe6f80b8c35701a8754bfe59d5 for an environment sample for Local (NOTE: environment variable names have been changed from the gist -- you must adjust accordingly)",
```
Comment (vimayya) [inline sample-api-calls/SEA Modernization Starter Kit.postman_collection.json]: @semalaiappan remove this line as local dev environment isn't needed in the kit

Comment (semalaiappan) [reply sample-api-calls/SEA Modernization Starter Kit.postman_collection.json]: Fixed it . Thanks

File: sample-api-calls/SEA Modernization Starter Kit Environment (Local).postman_environment.json (hunk 1 above)
Comment (semalaiappan) [reply sample-api-calls/SEA Modernization Starter Kit Environment (Local).postman_environment.json]: Fixed it . Thanks

File: sample-api-calls/SEA Modernization Starter Kit.postman_collection.json (hunk 2 above)
Comment (tmeador-dl) [reply sample-api-calls/SEA Modernization Starter Kit.postman_collection.json]: this still needs to be removed

**Review by tmeador-dl — CHANGES_REQUESTED** [review]

**Review by tmeador-dl — APPROVED** [review]

**Pull request merged**